### PR TITLE
Shopping cart

### PR DESCRIPTION
**Pull request by Sarahm458**

### CODE REVIEW

**Comment by Sarahm458** [comment]
im sorry i wrote checkout instead of shopping-cart

File: shopping-cart.html [hunk 1]
```diff
@@ -6,17 +6,99 @@
     <title>Shopping Cart Page</title>
     <link rel="stylesheet" href="./assets/CSS/all.css">
     <link rel="stylesheet" href="./assets/CSS/bootstrap.min.css">
+    <link rel="stylesheet" href="./assets/CSS/colors.css">
     <link rel="stylesheet" href="./assets/CSS/nav-footer.css">
     <link rel="stylesheet" href="./assets/CSS/shopping-cart.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
```
Comment (donia-elsayed) [inline shopping-cart.html]: please remove all these font-weights we need only medium 500

File: shopping-cart.html [hunk 2]
```diff
@@ -6,17 +6,99 @@
...
+                    <th scope="col">Price</th>
+                    <th scope="col" class="col-md-2">Quantity</th>
+                    <th scope="col">Subtotal</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="align-middle">
+                    <td><img src="assets/checkout-images/TV.png" alt="LCD Monitor" width="75" /> LCD Monitor</td>
+                    <td>$650</td>
+                    <td><button type="button" class="btn border-secondary text-center">+</button>
```
Comment (donia-elsayed) [inline shopping-cart.html]: please change the order of the buttons the minus button first then the plus button

File: shopping-cart.html [hunk 3]
```diff
@@ -6,17 +6,99 @@
...
+
+
+        <div class="container">
+            <div class="row">
+                <!-- Coupon code input -->
+                <div class="col-md-6 mb-3">
+                    <div class="input-group">
+                        <input type="text" class="form-control coupon" placeholder="Enter Coupon Code">
+                        <div class="input-group-append ml-2">
+                            <button class="btn btn-apply1" type="button">Apply Coupon</button>
```
Comment (donia-elsayed) [inline shopping-cart.html]: please change the button class btn-apply1 make it btn-apply only

File: shopping-cart.html [hunk 4]
```diff
@@ -6,17 +6,99 @@
...
+                            <div class="d-flex justify-content-between">
+                                <p class="card-text">Shipping:</p>
+                                <p class="card-text">Free</p>
+                            </div>
+                            <hr />
+                            <div class="d-flex justify-content-between">
+                                <p class="card-text">Total:</p>
+                                <p class="card-text">$200</p>
+                            </div>
+                            <button class="btn btn-success btn-block mt-3 btn-apply"><a href="checkout.html">Proceed to Checkout</a></button>
```
Comment (donia-elsayed) [inline shopping-cart.html]: please change  button class btn-apply to btn-toCheckout
also remove btn-success and btn-block

File: assets/CSS/shopping-cart.css [hunk 5]
```diff
@@ -1 +1,80 @@
...
+  color: white;
+  background-color: var(--secondary-color);
+  border: solid 2px var(--secondary-color);
+}
+
+
+
+
+
+* {
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: please move this code to the top and remove the fonstsize 14

File: assets/CSS/shopping-cart.css [hunk 6]
```diff
@@ -1 +1,80 @@
-@import 'colors.css';
\ No newline at end of file
+@import "colors.css";
+a {
+  text-decoration: none;
+  color: white;
+}
+.btn-apply1 {
+  color: var(--secondary-color);
+  font-weight: 500;
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: please remove all font-weight:500

File: assets/CSS/shopping-cart.css [hunk 7]
```diff
@@ -1 +1,80 @@
...
+}
+
+.btn-apply1:hover {
+  color: #4498e6;
+  background-color: white;
+  border: solid 2px #4498e6;
+
+}
+
+.btn-apply1::after {
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: why you repeat this code you can add btn-apply1:hover with btn-apply1

File: assets/CSS/shopping-cart.css [hunk 8]
```diff
@@ -1 +1,80 @@
...
+
+
+.coupon {
+  margin-right: 30px;
+  background-color: var(--bg-color);
+}
+
+
+
+.form-control:focus {
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: Please do not change property in the bootstrap  class directly you already have a class coupon

File: assets/CSS/shopping-cart.css [hunk 9]
```diff
@@ -1 +1,80 @@
...
+  font-weight: 500;
+  background-color: white;
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+
+
+
+.btn-apply {
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: you do not need to repeat the code you can make it once  using group selector

File: assets/CSS/shopping-cart.css [hunk 10]
```diff
@@ -1 +1,80 @@
...
+  color: white;
+  background-color: var(--secondary-color);
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+.btn-apply:hover {
+  color: white;
+  background-color: #4498e6;
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: please change the background color to --secondary-color

File: assets/CSS/shopping-cart.css [hunk 11]
```diff
@@ -1 +1,80 @@
...
+.btn-apply1 {
+  color: var(--secondary-color);
+  font-weight: 500;
+  background-color: white;
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+.btn-apply1:hover {
+  color: #4498e6;
```
Comment (donia-elsayed) [inline assets/CSS/shopping-cart.css]: please use the color schema we have, change the color and background color to --secondary-color

**Review by donia-elsayed — APPROVED** [review]

**Review by donia-elsayed — APPROVED** [review]

**Review by donia-elsayed — APPROVED** [review]

**Pull request merged**